GraphicEditor/Repositories/EquipmentRepository.cs

```diff
@@ -0,0 +1,63 @@
+﻿using GraphicEditor.Repositories.Interfaces;
+using Model.Hospital;
+using MySql.Data.MySqlClient;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GraphicEditor.Repositories
+{
+    public class EquipmentRepository : IEqipmentRepository
+    {
+        private MySqlConnection connection;
+        public EquipmentRepository()
+        {
+            connection = new MySqlConnection("server=localhost;port=3306;database=mydb;user=root;password=root");
+        }
+
+        private List<Equipment> GetEquipments(String query)
+        {
+            connection.Open();
+            MySqlCommand sqlCommand = new MySqlCommand(query, connection);
+            MySqlDataReader sqlReader = sqlCommand.ExecuteReader();
+            List<Equipment> resultList = new List<Equipment>();
+            while (sqlReader.Read())
+            {
+                Equipment entity = new Equipment();
+                entity.Name = (string)sqlReader[2];
+                entity.Id = (string)sqlReader[0];
+                entity.RoomId = (string)sqlReader[3];
+                resultList.Add(entity);
```

Comment:
We have serial numbers and don't need ids.
Columns need to be filled in right way other way we will not get what we want for every attribute.

Reply:
Equipment is inheriting Entity, which contain SerialNumber. Serial number will be placed at sqlReader[0] by default so you'll get serial number for id.